xorg-x11-drv-nvidia.spec

```diff
@@ -534,7 +552,9 @@ fi ||:
 %{_libdir}/libcuda.so.%{version}
 %{_libdir}/libnvcuvid.so.1
 %{_libdir}/libnvcuvid.so.%{version}
+%ifarch x86_64
```

Comment:
You are using %ifarch here, but ifnarch aarch64 earlier for the libnvidia-compiler.so.%{version} case
It seems better to assume %ifnarch aarch64 for both cases (to handle i686 libnvidia-compiler.so.%{version} that still exist).

Reply:
yep, fixed.